### PR TITLE
V katana2.1.x/feature/test report fixes

### PR DESCRIPTION
**Pull request by kkateq**
Please look at the jsontestresults.py method get_artifacts_path, not sure this is the correct approach to get the artifactServerURL. Do we have any way to get it directly from globalconfig here?

### CODE REVIEW

File: master/buildbot/status/web/jsontestresults.py [hunk 1]
```diff
@@ -23,9 +24,17 @@ def __init__(self, log, step_status):
         self.log = log
         self.step_status = step_status
 
+    def get_artifacts_path(self, build):
+        artifacts = build.artifacts.values() if build.artifacts else []
```
Comment (mariangemarcano) [inline master/buildbot/status/web/jsontestresults.py]: @kkateq the global configuration is not accessible here, and the artifact path is pass as a parameter in the upload download steps. let's check this together.

File: master/buildbot/test/unit/test_status_web_status_jsontestresults.py [hunk 2]
```diff
@@ -26,16 +26,24 @@
...
+                return self.artifactServerPath
+            return None
+
         st = self.build_step_status = mock.Mock(BuildStepStatus)
 
         build_status = mock.Mock(BuildStatus)
         build_status.getNumber = lambda: 1
+
         builder_status = mock.Mock(BuilderStatus)
         builder_status.getFriendlyName = lambda: "BuilderStatusFriendlyName"
```
Comment (campos-ddc) [inline master/buildbot/test/unit/test_status_web_status_jsontestresults.py]: By the way, all of these could also be written as:

`build_status.getFriendlyName.return_value = "BuilderStatusFriendlyName"`

File: www/templates/jsontestresults.html [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 {% block suites %}
-
+   <div class="content container">
     {% for suite in data.suites %}
         {% if suite.summary.result %}
             <div class="row">
-                <h2>Suite type: {{ suite.name }}</h2>
-                {% if data.utr_prefix %}
+                <h4>Suite type: {{ suite.name }}</h4>
+                {% if data.utrPrefix %}
```
Comment (opdude) [inline www/templates/jsontestresults.html]: Is this the correct name for the variable name now? utrPrefix? I don't see where the change is on the data side

Comment (kkateq) [inline www/templates/jsontestresults.html]: This is json property

**Comment by opdude** [comment]
Front end changes look fine :)

**Pull request merged**